example/server/apps.py

```diff
@@ -6,8 +6,10 @@ class AnchorConfig(AppConfig):
 
     def ready(self):
         from polaris.integrations import register_integrations
-        from .integrations import MyDepositIntegration, MyWithdrawalIntegration
+        from .integrations import (MyDepositIntegration,
```

Comment:
I've noticed a lot of formatting changes in the PR, can we do a PR thats just running the formatter on all the files, with no other changes, so we don't have unrelated changes in other PRs?

Reply:
Sure. I just added a pre-commit hook for this repo to use `black` formatting.